pkg/testcase/ingressdns.go

```diff
@@ -99,7 +102,7 @@ func TestIngressRoute(applyWorkload, deleteWorkload bool, apiVersion string) {
 
 	getIngressRoutePodsRunning := fmt.Sprintf("kubectl get pods -n test-ingressroute -l app=whoami"+
 		" --field-selector=status.phase=Running --kubeconfig=%s", shared.KubeConfigFile)
-	err := assert.ValidateOnHost(getIngressRoutePodsRunning, statusRunning)
+	err = assert.ValidateOnHost(getIngressRoutePodsRunning, statusRunning)
 	Expect(err).NotTo(HaveOccurred(), err)
 
 	// Query the IngressRoute Host
```

Comment:
since we are touching here again and i forgot this on the original PR, can we extract here unto line 133 into a separated func?